### PR TITLE
Updating user stories

### PR DESCRIPTION
**Pull request by dmurko**
Ref https://github.com/niteoweb/operations/issues/660

### CODE REVIEW

File: 2_Operations/user-stories.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ All major features should have a blog post written. If you are not sure if it sh
 
 The expected outcomes of the user story which is a list of items stating how the world will look once the user story has been completed. This is not the individual task or activities but rather what will the documentation look like, what new tests will we see, what bug no longer occurs because it has been fixed.
 
-## Story points
+## Story Points
 
 A [story point](https://agilefaq.wordpress.com/2007/11/13/what-is-a-story-point/) is an arbitrary measure used by Scrum teams to indicate the effort required to implement a User Story. One full-time member on sprint should be able to do 10 Story Points.
```
Comment (marbeann) [inline 2_Operations/user-stories.md]: `User Story` -> `user story` for consistency

File: 2_Operations/user-stories.md [hunk 2]
```diff
@@ -61,9 +57,9 @@ The Scrum master and product owner will verify that the user story is well defin
 
 #### Voting
 
-Once the user story is verified the Scrum master assigns team members from whom they want to receive [story point](#story-points) estimations.
+Once the user story is verified the Scrum master assigns team members from whom they want to receive story point estimations. Team members vote on a story by adding a comment with their story point estimation to the user story, then remove their assignment from the user story.
 
-Team members vote on a story by adding a comment with their [story point](#story-points) estimation to the user story, then remove their assignment from the user story.
+A Nitean who is full-time on a sprint should be able to finish 10 story points.
```
Comment (zupo) [inline 2_Operations/user-stories.md]: This is an exact duplicate of the sentence above in line 38.

File: 2_Operations/user-stories.md [hunk 3]
```diff
@@ -1,29 +1,23 @@
...
-Some frequently used personas:
-
- * Nitean
- * Nitean on support
- * Developer
- * Marketer
- * DevOp
-
-An end-user story should be written using the product's personas.
+Whenever possible, think about the end user (product persona) and how the result impacts them. Other possible personas are Nitean, Support, Developer, Marketer and DevOps. That said, very rarely is something not done directly for the user.
```
Comment (karantan) [inline 2_Operations/user-stories.md]: Can we have a link to those personas defined in intra

Comment (dmurko) [reply 2_Operations/user-stories.md]: Yes. WooCart has them in project repo, EBN will have too soon.

**Pull request merged**